package/windows/Dockerfile

```diff
@@ -0,0 +1,10 @@
+ARG SERVERCORE_VERSION
+
+FROM mcr.microsoft.com/powershell:nanoserver-${SERVERCORE_VERSION}
```

Comment:
if all youre doing in is `wins.exe --version` you can skip using pwsh or servercore... just use `mcr.microsoft.com/windows/nanoserver:` and set the `CMD ["wins.exe", "--version"]`. can have 1 docker file then and bypass the powershell wrapper. agree?

Reply:
We need Powershell since the actual script that will be run on this image is [upgrade.ps1](https://github.com/rancher/wins/blob/8b4d2f4aaf0cd01f8dfc6544703dd449c0c3176e/charts/rancher-wins-upgrader/scripts/upgrade.ps1) as this is meant to be a Docker image that will be used by the upgrade chart.

As discussed offline, I'll be modifying the Dockerfile to use servercore and only have 1 docker image for 20H2 + older versions.

Reply:
Addressed in latest commit.